src/xenia/emulator.cc

```diff
@@ -196,6 +196,8 @@ X_STATUS Emulator::Setup(
   // Bring up the virtual filesystem used by the kernel.
   file_system_ = std::make_unique<xe::vfs::VirtualFileSystem>();
 
+  patching_system_ = std::make_unique<xe::patcher::PatchingSystem>();
```

Comment:
The word "system" is redundant, as most large classes are systems in general.

Reply:
I would agree with Triang3l that using system in the name is a bit unclear, perhaps something like `PatchDB` would make more sense?